apps/guardian/src-tauri/src/main.rs

```diff
@@ -1188,24 +1188,20 @@ async fn generate_decryption_proofs(
         .context("Could not find encrypted tally in app state")?;
 
     // Find all decryption shares for all guardians. If the shares registered by a
-    // specific guardian are either missing or cannot be decoded, return
-    // `Error::DecryptionShareError`.
-    let decryption_shares: Vec<_> =
-        contract_data
-            .guardians
-            .iter()
-            .map(|(_, guardian_state)| {
-                let bytes = guardian_state.decryption_share.as_ref().ok_or(
-                    Error::InvalidDecryptionShare(
-                        "Not all selected guardians posted their decryption shares".into(),
-                    ),
-                )?;
-                let shares = decode::<GuardianDecryption>(bytes).map_err(|_| {
-                    Error::InvalidDecryptionShare("Invalid decryption shares were detected".into())
-                })?;
-                Ok::<_, Error>(shares)
-            })
-            .try_collect()?;
+    // specific guardian cannot be decoded, return
+    // `Error::DecryptionShareError`. If the shares are missing, exclude them from
```

Comment:
Isn't the error `InvalidDecryptionShare`?

Reply:
I like to keep you on your toes.. good catch!